docs/accessibility/configuration/attributefilters.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: 'Attribute Filters | Cypress Accessibility'
+description: 'The `attributeFilters` configuration property allows users to specify patterns for attributes and their values that should not be used for identifying and grouping elements.'
+sidebar_label: attributeFilters
+sidebar_position: 50
+---
+
+<ProductHeading product="accessibility" /># attributeFilters
```

Comment:
This is not showing up right.

![Screenshot 2025-06-02 at 11 23 04 AM](https://github.com/user-attachments/assets/2591b55c-f24f-4f77-81f0-2ed61c873440)